dataBase.js

```
@@ -1,29 +1,25 @@
 const { Sequelize } = require('sequelize');
 
-const sequilize = new Sequelize({
+const sequelize = new Sequelize({
   dialect: 'sqlite',
-  storage: './db.sqlite3'
+  storage: './db.sqlite3',
+  logging: false,
 });
 
 async function clear() {
-  const t = await sequilize.transaction();
-
   try {
-    await sequilize.query('DELETE FROM Article;');
-    await sequilize.query('DELETE FROM User;');
-    await sequilize.query('DELETE FROM Tag;');
-    await sequilize.query('DELETE FROM ArticleTag;');
-    await sequilize.query('DELETE FROM UserFollowUser;');
-    await sequilize.query('DELETE FROM UserFavoriteArticle;');
-    await sequilize.query('DELETE FROM Comment;');
-
-    await t.commit();
+    await sequelize.query('DELETE FROM User;');
```

Table name correctness: you issue SQL against `User`, `Article`, etc. Sequelize often pluralizes model names (e.g. `Users`), or table names could be lowercase. If the real DB tables have different names/casing, these DELETE statements will silently fail (no rows deleted). I recommend verifying actual table names (e.g. using `sequelize.getQueryInterface().showAllTables()` or reading `sequelize.models`) or switching to QueryInterface/bulkDelete with model table names to avoid mismatches.